dockerfiles/entrypoints/celery.sh

```diff
@@ -2,7 +2,11 @@
 
 ../../docker/common.sh
 
-python3 ../../docker/scripts/wait_for_search.py
+# If SEARCH is enabled, the celery container should wait for the search
+# container to be ready
+if [ "$SEARCH" != "" ]; then
+  ../../docker/wait-for-it.sh search:9200 --timeout=360 --strict
```

Comment:
6 minutes? That seems like a _lot_ 🤯

Reply:
Mine just took 121 seconds to start, so I didn't really know what would be a good timeout.. perhaps more like 240 seconds? :)

Reply:
Reduced to 180 secs